assets/src/hooks/useRouteFilter.tsx

```diff
@@ -58,7 +58,11 @@ export const RouteFilter = ({
         />
 
         {filterText.length > 0 ? (
-          <button className="m-route-filter__clear" onClick={clearTextInput}>
+          <button
+            className="m-route-filter__clear"
+            onClick={clearTextInput}
+            data-testid="clear-text"
```

Comment:
Could this be a title instead, or is there a reason that wouldn't be appropriate here?